setup.py

```diff
@@ -42,8 +42,9 @@ def _guard_py_ver():
     cur_py = parse('.'.join(map(str, sys.version_info[:3])))
 
     if not min_py <= cur_py < max_py:
-        msg = ('Cannot install on Python version {}; only versions >={},<{} ',
-               'are supported.')
+        msg = (
+            'Cannot install on Python version {}; only versions >={},<{} are supported.'
+        )
```

Comment:
This line needs wrapping as it exceeds 80 chars. From the original message, I think the comma simply needs removing and it'll work as intended?

Reply:
Yes, my bad, I wasn't using my usual IDE setup so the flake8 config didn't get picked up, will fix now.